data/calculated_parser/functions.py

```diff
@@ -9,6 +9,7 @@
 import xarray as xr
 from metpy.units import units
 from pint import UnitRegistry
+import time
```

Comment:
Is anything from `time` being used in this file?

Reply:
removed